.schemastore/config.schema.json

```diff
@@ -126,6 +126,60 @@
         "config"
       ]
     },
+    "httpRequestConfig": {
+      "type": "object",
+      "properties": {
+        "url": {
```

Comment:
Any ideas on how to make the URL required only if the "courier.http" object is set?

Reply:
Yes, first make "url" a required field in the object. Then you need to do an if statement in the courier object. i think there's some examples in webhooks how this works.